### ISSUE TITLE
`api.Permissions.permission_accessibility-events` - What is this?

### ISSUE THREAD

**Issue by ddbeck** [mdn/browser-compat-data#25854]
### What type of issue is this?

Missing specification link (no spec_url property)

### What information was incorrect, unhelpful, or incomplete?

Narrowly, `api.Permissions.permission_accessibility-events` doesn't have a specification. More generally, the data itself and the commit history provides no clues as to what API this permission is associated with or whether the permission actually exists under this or any other name.

### What did you expect to see?

A specification URL.

### Did you test this? If so, how?

No.

### Can you link to any release notes, bugs, pull requests, or MDN pages related to this?

This is the history of the non-automated changes I could find for this data:

In https://github.com/mdn/browser-compat-data/pull/2668/, the permission was originally added to BCD under the key `accessibility-events_permission` and description "<code>accessibility</code> events permisson".

In https://github.com/mdn/browser-compat-data/pull/5909/, the description changed to "<code>accessibility-events</code> events permisson". There was no discussion about this—it was presumed to be correct, I guess. This looks awfully suspicious to me; the "permisson" typo was preserved in the description rewrite.

In https://github.com/mdn/browser-compat-data/pull/18561, the key was renamed to `permission_accessibility-events`, to order it with other permission names.

I tried searching webref for a plausibly connected spec, but since "accessibility" and "events" are perhaps two of the most common topics across specs, my search was fruitless.

What is this thing and where did it come from?

**Comment by caugner** [comment]
The permission was added in Chrome 62 via https://github.com/chromium/chromium/commit/72947e01e2f423f13183457047bb7c8d763e1893, see: https://chromiumdash.appspot.com/commit/72947e01e2f423f13183457047bb7c8d763e1893

And it looks like it was removed in Chrome 131 via https://github.com/chromium/chromium/commit/41da4aa10c42b36665b50c076274ab0dff0261e3, see: https://chromiumdash.appspot.com/commit/41da4aa10c42b36665b50c076274ab0dff0261e3

**Comment by ddbeck** [comment]
I've sent https://github.com/mdn/browser-compat-data/pull/25860/ to update the support info and resolve the specification question. See that PR for a longer discussion.
